pkg/reconciler/revision/table_test.go

```diff
@@ -599,6 +621,18 @@ func withK8sServiceName(sn string) RevisionOption {
 	}
 }
 
+func withPaLastPinned(t time.Time) PodAutoscalerOption {
+	return func(pa *autoscalingv1alpha1.PodAutoscaler) {
+		if pa.ObjectMeta.Annotations == nil {
+			pa.ObjectMeta.Annotations = make(map[string]string)
```

Comment:
```suggestion
			pa.ObjectMeta.Annotations = make(map[string]string,1)
```
We know the future size.

Reply:
> We know the future size.

That would not be correct. Further annotations could be added, eg. with `WithKPAClass` or `WithHPAClass`.

Reply:
It sets the min size, not the max, so it's a performance hint